superset/assets/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -122,7 +124,7 @@ class SliceHeaderControls extends React.PureComponent {
         </Dropdown.Toggle>
 
         <Dropdown.Menu>
-          <MenuItem onClick={this.refreshChart}>
+          <MenuItem onClick={this.refreshChart} disabled={!updatedDttm}>
```

Comment:
`updatedDttm` is a copy of chart's `chartUpdateEndTime`. it will be reset to null when a `CHART_UPDATE_STARTED` is triggered. it is pretty much same way as we reset `chartStatus`, but since this attribute is already passed down to component, i don't have to introduce another 
 props.